aws_attach_integration_permissions/attach_integration_permissions.py

```diff
@@ -0,0 +1,131 @@
+import json
```

Comment:
left this in for easier review but it's not really necessary, can remove before merging